unittests/runtime/LongTests/LongRefcounting.cpp

```diff
@@ -142,54 +142,194 @@ static void releaseALot(TestObject *object, size_t &deallocated,
 const uint64_t maxRC = 1ULL << (32 - 2);
 
 TEST(LongRefcountingTest, retain_max) {
-  size_t deallocated = 0;
-  auto object = allocTestObject(&deallocated, 1);
+  // Don't generate millions of failures if something goes wrong.
+  ::testing::FLAGS_gtest_break_on_failure = "true";
```

Comment:
I'm seeing warnings due to this and a handful of other lines like it. It appears that FLAGS_gtest_break_on_failure is a bool rather than a string.

warning: implicit conversion turns string literal into bool: 'const char [5]' to 'bool' [-Wstring-conversion]
  ::testing::FLAGS_gtest_break_on_failure = "true";